layouts/wrapper.html

```diff
@@ -27,6 +27,16 @@
 		<meta name="msapplication-TileColor" content="#fff1e0">
 		<meta name="msapplication-TileImage" content="https://next-geebee.ft.com/assets/brand-alpha/icons/mstile-144x144.png">
 		<meta name="theme-color" content="#fff1e0">
+		
+		{{#if og}}
+		<meta property="og:title" content="{{ og.title }}"/>
+		<meta property="og:url" content="{{ og.url }}"/>
+		<meta property="og:description" content="{{ og.description }}"/>
+		<meta property="og:image" content="{{ og.image }}"/>
+		<meta property="og:type" content="article"/>
```

Comment:
any reason why this stuff is here (particularly the `article` bit?  potentially all could live inside `next-article` making use of the head output block helper……?